source/views/dictionary/list.tsx

```diff
@@ -53,12 +53,22 @@ const styles = StyleSheet.create({
 	},
 })
 
-type Props = TopLevelViewPropsType
-
-export function DictionaryView(props: Props) {
+function DictionaryView(): JSX.Element {
```

Comment:
Thoughts on `JSX.Element` vs `React.ReactNode`? I'd rather go with whatever precedent we have, but I'm not sure what it is :)

Reply:
This is what I have been relying on as far as a [comprehensive answer](https://stackoverflow.com/a/59840095).

> Why do the render methods of class components return `ReactNode`, and function components return `ReactElement`.
> 
> It's a TS type incompatibility.
> 
> * TS class component returns `ReactNode` with `render()`
> * TS function component returns `JSXElement | null`